### DISCUSSION TITLE
Number Recommender - Tables & Disclosures

### DISCUSSION THREAD

**Discussion by kdocherty6** [twilio-labs/paste#3111] [Office hours] [answered]
### Background

We are creating a new experience within Phone Numbers to help customers through the decision process of choosing the right number type for their use case. This Phase 1 version will be limited and will focus on showing them the available numbers for their use case and providing more information on the steps to provision those numbers.

### Stage of design

High-fidelity

### Research

Discovery Research - https://docs.google.com/presentation/d/1ZWGwXxgzqXmMF93NeCiBz-RQ9foKcuXsEH6Iv9wh650/edit 

### Relevant links

Figma - https://www.figma.com/file/siTZC3BJDHRVLjzXel00ZA/PN--Number-Recommender?node-id=1579%3A454075&t=2VXAq0U8nFsIxEmq-1

### Are you looking for visual design support?

- [X] Yes

### Additional Information

I am struggling with how to best show the information on the results page. Given the user can select up to 5 countries and have multiple number options per country, I was thinking of using disclosures to contain the results for each country but tables within those are getting messy. Looking for any suggestions to better display this information

**Comment by SiTaggart** [comment]
Hey @kdocherty6 

Thanks for signing up and providing great pre reads. Look forward to seeing you in office hours

**Comment by SiTaggart** [comment, marked as answer]
Hi @kdocherty6 

The recording from office hours can be found here https://www.loom.com/share/ad68f125c26446d0a8c6a4beb480112b and the passcode is found in the #help-design-system slack channel.
